mappings/net/minecraft/entity/projectile/FishingBobberEntity.mapping

```diff
@@ -1,10 +1,17 @@
 CLASS net/minecraft/class_1536 net/minecraft/entity/projectile/FishingBobberEntity
 	CLASS class_1537 State
 	FIELD field_7165 hookedEntity Lnet/minecraft/class_1297;
-	FIELD field_7168 luckOfTheSeaLevel I
+	FIELD field_7166 selfHitTimer I
+	FIELD field_7167 removalTimer I
+	FIELD field_7168 lureLevel I
+	FIELD field_7169 fishAngleDegrees F
```

Comment:
`degrees` may be unnecessary as most minecraft angles are in degrees

Reply:
fair point.